Modules/_sqlite/connection.c

```diff
@@ -1998,8 +1998,9 @@ pysqlite_connection_iterdump_impl(pysqlite_Connection *self)
         }
         return NULL;
     }
-
-    PyObject *retval = PyObject_CallOneArg(iterdump, (PyObject *)self);
+    PyObject *py_filter = NULL;
+    py_filter = PyUnicode_FromString(filter);
+    PyObject *retval = PyObject_Call(iterdump, (PyObject *)self, py_filter);
```

Comment:
You can use vector call for this (you'll find examples for how to do that a few other places in `connection.c`, IIRC).

Reply:
Finally figured out how to use `PyObject_Vectorcall()` :tada: